app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,51 @@
+class CommentsController < ApplicationController
+  #index
+  def index
+  @post = Post.find(params[:post_id])
+  @comment = Comment.all
```

Comment:
Ha and now that I think about it, you're not using this route/controller action anyways, so doesn't make much of a difference either way!